ansible-cli.js

```diff
@@ -23,12 +23,20 @@ async function execute({
     ...curr.environmentVariables,
   }), {});
 
-  const updatedParams = params;
+  const updatedParams = _.cloneDeep(params);
   if (volumeConfigsMap.has("vaultPasswordFile")) {
     updatedParams.vaultPasswordFile = `$${volumeConfigsMap.get("vaultPasswordFile").mountPoint}`;
   }
 
-  const ansibleCommand = createAnsibleCommand(command, updatedParams, additionalArguments);
+  const ansibleCommand = createAnsibleCommand(
+    command,
+    {
+      playbookName: updatedParams.playbookName,
+      sshPassword: updatedParams.sshCredentials.password,
+      vaultPasswordFile: updatedParams.vaultPasswordFile,
```

Comment:
Now its visible that only three of the original params are required here, and only one of them might be mutated. In such case `clone` is an overkill, and the intent can be clearly expressed with a creation of an object which:
- Has three params, read from the original object.
- With one of the params value set conditionally
I conclude that the object will contain `ansibleCommandParams`